### DISCUSSION TITLE
[closed] Fingerabdruck-Datenbank

### DISCUSSION THREAD

**Discussion by 47441** [frickelzeugs/FingerprintDoorbell#55] [Ideas]
Hallo @frickelzeugs,

vielen Dank für Deine Idee und die Umsetzung. Ich habe es noch nicht bei mir umgesetzt, bin aber in der Überlegung und habe die Hardware bereits bestellt. ;)

Wir überlegen einen Vereinsraum mit diesem System auszustatten. Dazu sollen rund 150 Fingerabdrücke eingelesen werden, die berechtigt sind, die Tür zu öffnen.

Gibt es eine Möglichkeit das Anlernen der Fingerabdrücke an einem anderen System durchzuführen als an dem "Livesystem" an der zu schließenden Tür? Kann die Datenbank von einem auf ein anderes System kopiert werden? 

Vielen Dank

Martin

**Comment by frickelzeugs** [comment]
Hi @47441,

die Muster zur Fingerabdruck-Erkennung werden auf dem Sensor selbst gespeichert. Aktuell gibt es keine Möglichkeit diese Daten von einem Sensor auf den anderen zu kopieren. Das Anlernen muss daher immer am Sensor direkt durchgeführt werden. Im Falle eines Sensor-Tauschs (z.B. bei Defekt) muss daher das komplette Prozedere am neuen Sensor wiederholt werden. Bei einer kleinen Familie ist das noch kein großes Problem, aber alle 150 Personen des Vereins im Fehlerfall nochmal antanzen zu lassen...

Ich würde daher von deinem Vorhaben abraten und auf ein fertiges (kommerzielles?) System setzen, dass für solche Szenarien besser geeignet ist, z.B. ein System, dass die Fingerabdruck-Datenbank übertragbar macht oder eine Zugangskontrolle über RFID-Karten/Tokens/NFC-Smartphone anbietet.

**Reply by 47441** [reply]
Hallo @frickelzeugs,

vielen Dank. Ich hatte es schon vermutet/befürchtet. Andererseits macht es das ja auch sicherer, dass man es nicht so leicht manipulieren kann. ;)
An der Suche nach Alternativen wind wir schon dran. 
Herzliche Grüße und ein besinnliches und gesegnetes Weihnachtsfest.
Martin
